### ISSUE TITLE
Domain: Creates a guest account in Azure Active Directory (AD).

### ISSUE THREAD

**Issue by nhogarth86** [MicrosoftDocs/memdocs#1375]
For the Guest account section it says "Domain: Creates a guest account in Azure Active Directory (AD)." - is this correct? From my testing it just allows an Azure AD account to login to the device, it doesn't create a guest account in Azure AD.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fe42a212-bb45-2823-80e7-584df7898e53
* Version Independent ID: 969f46bf-e963-fc1f-22b8-78c18df19098
* Content: [Windows 10 shared device settings - Microsoft Intune - Azure](https://docs.microsoft.com/en-us/mem/intune/configuration/shared-user-device-settings-windows)
* Content Source: [memdocs/intune/configuration/shared-user-device-settings-windows.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/configuration/shared-user-device-settings-windows.md)
* Service: **microsoft-intune**
* Sub-service: **configuration**
* Technology: ****
* GitHub Login: @MandiOhlinger
* Microsoft Alias: **mandia**

**Comment by MandiOhlinger** [comment]
@nhogarth86 Good catch. Thank you! We updated the descriptions in the article. Hopefully the changes will be live this week.
